### ISSUE TITLE
Support `export * as _` syntax

### ISSUE THREAD

**Issue by johnnyshankman** [qmhc/vite-plugin-dts#320]
### Description

As a developer, many plugins etc autogenerate files which we have no control over. Some of them use `export * as` syntax. This plugin does not support that syntax, therefore it's useless in those scenarios.

### Suggested solution

Support the `export * as __` syntax.

### Additional context

```
Error: [vite:dts] The "export * as ___" syntax is not supported yet; as a workaround, use "import * as ___" with a separate "export { ___ }" declaration
```

Ran into this using typechain

https://github.com/dethcrypto/TypeChain/issues/900

### Validations

- [X] Read the [FAQ](https://github.com/qmhc/vite-plugin-dts#faq).
- [X] Check that there isn't [already an issue](https://github.com/qmhc/vite-plugin-dts/issues) that reports the same bug to avoid creating a duplicate.

**Comment by qmhc** [comment]
This seems to be a limitation of `@microsoft/api-extractor`.

**Comment by JeanBarriere** [comment]
This seems to be fixed by https://github.com/microsoft/rushstack/blob/main/apps/api-extractor/CHANGELOG.md#7470

**Comment by itwmike** [comment]
作者为啥不升级呢？这是个很严重的功能问题

**Comment by qmhc** [comment]
Fix by 7a5fe9a55d8503f4af88e935f4b7d3399a4a832e.

**Comment by johnnyshankman** [comment]
@qmhc @JeanBarriere thank you!
